backend/src/main/java/ca/bc/gov/hlth/hnweb/security/TransactionType.java

```diff
@@ -17,7 +17,8 @@
 	CANCEL_GROUP_MEMBER("CancelGroupMember"),
 	CANCEL_DEPENDENT("CancelDependent"),
 	GET_CONTRACT_PERIODS("GetContractPeriods"),
-	CONTRACT_INQUIRY("ContractInquiry"),
+	CONTRACT_INQUIRY("ContractInquiry"),	//R40
+	GET_GROUP_MEMBERS_CONTRACT_ADDRESS("GetGroupMembersContractAddress"), //R37
```

Comment:
We should just use GET_CONTRACT_ADDRESS(GetContactAddress) which matches the new terminology: GetContractAddress. The list of transactions is available on the Audit Design page.